### DISCUSSION TITLE
Use fasta database from NCBI

### DISCUSSION THREAD

**Discussion by BenoitBe** [vdemichev/DiaNN#424] [Q&A]
Hello Vadim,

I am performing a research wih proteomes from NCBI, like this:

>NP_001289901.1 uncharacterized protein LOC104746256 [Camelina sativa]

Actually, it not work directly with DIA-NN (see only 21 entries).

Do I have to specifiy something?

Many thanks in avance,

Benoit

**Comment by vdemichev** [comment]
Hi Benoit,

Can you please describe (attach here) the output you see, indicate what would you expect and point to discrepancies?

Best,
Vadim

**Comment by BenoitBe** [comment]
Dear Vadim,

I use fasta file like this:

![image](https://user-images.githubusercontent.com/90833041/177152000-7537eb10-f7d1-4b15-b7c8-213d6e3e20a0.png)

And here is the log:

[0:25] Processing FASTA
[0:56] Assembling elution groups
[1:36] 9587732 precursors generated
[1:36] Gene names missing for some isoforms
[1:36] Library contains 21 proteins, and 21 genes
[1:37] Encoding peptides for spectra and RTs prediction
[2:08] Predicting spectra and IMs
[131:40] Predicting RTs
[150:46] Decoding predicted spectra and IMs
[150:54] Decoding RTs

I know that DIA-NN support UniProt fasta file format, is it possible from NCBI format?

Thx in advance,

Benoit

**Reply by BenoitBe** [reply]
My fasta file has 107481 "genes", mainly annotated >XP

**Reply by vdemichev** [reply]
OK, by 'proteins' and 'genes' it means uniprot protein names and gene names. So these are apparently not recognised (as this is not uniprot). But all sequence IDs and the respective peptides were correctly added to the library.

**Reply by BenoitBe** [reply]
But not seem to work at the first passage, I have only: 5 (precursor-level), 0 (protein-level)

Despite I have nice signal on MS.

**Reply by BenoitBe** [reply]
And no, this is not UniProt, this is from Genbank.

**Reply by vdemichev** [reply]
You will see protein ids in the main report

**Reply by BenoitBe** [reply]
Ok, thanks, I will perform the full research and give you the feedback.

Regards, 

Benoit
